content/en/blog/2023/announcing-notation-rc4.md

```diff
@@ -0,0 +1,124 @@
+}
+```
+
+### Support experimental features
+
+Experimental features are intended for testing and evaluation purposes only and should not be used in production environments. You can now enable experimental features in Notation CLI by setting the `NOTATION_EXPERIMENTAL` environment variable to `1`. Here's an example of how to set the environment variable in Linux or macOS:
+
+```shell
+export NOTATION_EXPERIMENTAL=1
+```
+
+And here's an example of how to set the environment variable in Windows PowerShell:
+
+```powershell
+$env:NOTATION_EXPERIMENTAL = "1"
+```
+
+Once you've set the environment variable, you can use Notation with experimental features enabled. Here is the list of experimental features that Notation introduced:
+
+- Store signatures using artifact manifest (Require Registry support)
+- Signing and verifying images as OCI image layout
+
+### Use notation policy command to manage trust policy configuration
+
+To simplify the user experience of managing trust policy configuration, we have introduced two new commands in the v1.0.0-rc.4 release: `notation policy import` and `notation policy show`.
+
+Previously, users had to follow several steps to configure trust policy configuration in a JSON file, save the file with a specific name, ensure the file encoding was utf-8, and put the file in a specific directory that varied across different operating systems. With the `notation policy import` command, users can now import trust policy configuration directly from the JSON file after completing the first step above. Additionally, a health check is performed on the trust policy configuration during import to avoid configuration issues that may arise later. For example:
+
+```shell
+notation policy import mypolicy.json
+```
+
+The `notation policy show` command allows users to easily view trust policy configuration and redirect the output to a file for sharing, updating, or other purposes. For example:
+
+```shell
+notation policy show
+```
+
+We will be introducing more commands in future releases. Stay tuned for updates on how we're making it easier for you to manage your trust policy configuration.
+
+### Support OCI distribution 1.1 referrers API
+
+The Referrers API is a new feature added in [OCI distribution spec v1.1-rc.1](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#listing-referrers), which allows clients to fetch a list of referrers in an efficient and clean manner. In the context of Notation, referrers are signatures that refer to the container image. Since this release, Notation verifies whether the Referrers API is available in the registry when pushing signatures into the registry. If the Referrers API is not available, Notation follows the [fallback procedure](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md#unavailable-referrers-api) and updates the image index pushed to a tag described by the referrers tag schema.
+
+### Sign and verify images as OCI image layout
```

Comment:
Same as above

Reply:
Need a short version for title, any other proposal?